directus/version.js

```diff
@@ -0,0 +1,9 @@
+import { readFileSync } from 'node:fs';
+import { dirname, resolve } from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+const pkg = JSON.parse(readFileSync(resolve(__dirname, 'package.json'), 'utf8'));
```

Comment:
While we're at it, can we make it async? 🙏
Also no need for `resolve` since `__dirname` is already absolute.

```suggestion
import { readFile } from 'node:fs/promises';
import { dirname, join } from 'node:path';
import { fileURLToPath } from 'node:url';

const __dirname = dirname(fileURLToPath(import.meta.url));

const pkg = JSON.parse(await readFile(join(__dirname, 'package.json'), 'utf8'));
```